docs/usage/noise-reduction.md

```diff
@@ -98,8 +98,8 @@ Or perhaps at least weekly:
   ]
 ```
 
-If you're wondering what is supported and not, under the hood, the schedule is parsed using [later.js](https://bunkat.github.io/later/) using the `later.parse.text(scheduleString)` API.
-[This page](https://bunkat.github.io/later/parsers.html#text) explains the supported syntax or you can experiment on the [RunKit playground](https://npm.runkit.com/later).
+If you're wondering what is supported and not, under the hood, the schedule is parsed using [@breejs/later](https://github.com/breejs/later) using the `later.parse.text(scheduleString)` API.
+TODO INSERT PROPER LINK HERE explains the supported syntax DROPPED LINK TO RUNKIT PLAYGROUND.
```

Comment:
We used to have a link to some `parsers.html#text` docs, but I don't know what the replacement should be on the new https://breejs.github.io/later/getting-started.html docs. So if you code wizards can find out what the proper replacement is, that would be great! 😄 😉 

I dropped the link to the `runkit playground`, don't know if we need a replacement or not.

Reply:
https://breejs.github.io/later/parsers.html#text 🙃 

Maybe we should clarify that we don't support schedules which are smaller that hourly as renovate will only run hourly (or on hooks for app and on premise)